diagrams/cs-for-hackers.js

```diff
@@ -0,0 +1,117 @@
+var nameSpace = 'template';
+
+var svgWidth  = 250;
+var svgHeight = 250;
+
+var xOffset = 40;
+
+
+// dynamic values
+
+var textColor = '#555555';
```

Comment:
you don't need this. text is this color by default.

so in the generator functions:

```
generateText    ( content, x, y, fontSize, [ fontWeight, fill, fontFamily, textAnchor, transform, style, mask ])
```

anything in brackets are optional. so only content, x, y, and fontSize are required. everything else gets a default, which you can see in default_values.js

also, don't use the hex color '#555', because we already have a variable for that, `darkColor`, and you can just use that variable in your file. This way your diagrams are future-proofed. If we ever change our dark color, all we have to do is change the value of `darkColor` instead of going through and finding all the places where we typed out '#555'